ReactCommon/hermes/executor/HermesExecutorFactory.h

```diff
@@ -36,6 +40,8 @@ class HermesExecutorFactory : public JSExecutorFactory {
   JSIExecutor::RuntimeInstaller runtimeInstaller_;
   JSIScopedTimeoutInvoker timeoutInvoker_;
   ::hermes::vm::RuntimeConfig runtimeConfig_;
+  bool enableDebugger_ = true;
+  std::string debuggerName_ = "Hermes React Native";
```

Comment:
We default in two locations: 
1. In HermesExecutorFactory.java
2. here.

Can we keep the defaults in HermesExecutorFactory.java?

Reply:
since HermesExecutorFactory.h is also be used on ios, i'd set the default here.

also in HermesExecutorFactory.java the default value is empty and we use cpp's default if the the value from jni is empty. so we don't have to maintain the "Hermes React Native" string in two places.